lib/fastlane/plugin/ddg_apple_automation/actions/tag_release_action.rb

```diff
@@ -35,7 +34,7 @@ def self.run(params)
           return
         end
 
-        if params[:ignore_untagged_commits]
+        if should_merge_before_deleting(params)
```

Comment:
This is an unrelated improvement – only enter this for public releases (to guard from the case when someone sets this option for an internal release).